public/src/components/campaign/campaign_list.jsx

```diff
@@ -15,21 +15,15 @@ export default class CampaignList extends Component {
   render() {
     const {
       account_info,
-      addCampaignToUser,
       all_campaigns,
       auth,
       history,
-      setCurrentCampaign
+      updateCampaignStatus
```

Comment:
Why did you remove setCurrentCampaign? I think you're using this in your current ticket, right?

Reply:
hmmm maybe just comment it out?

Reply:
Addressed per our conversation